### ISSUE TITLE
Doc path breaks on Macs if exe is renamed (Trac #408)

### ISSUE THREAD

**Issue by butlerpd** [SasView/sasview#542]
If one renames the SasView program name (e.g. SasView2) on the mac all the documentation opens blank pages.  This is not true for PC.

Migrated from http://trac.sasview.org/ticket/408
```json
{
    "status": "closed",
    "changetime": "2015-11-22T01:09:16",
    "_ts": "2015-11-22 01:09:16.090192+00:00",
    "description": "If one renames the SasView program name (e.g. SasView2) on the mac all the documentation opens blank pages.  This is not true for PC.",
    "reporter": "butler",
    "cc": "",
    "resolution": "invalid",
    "workpackage": "SasView GUI Enhancements",
    "time": "2015-04-08T03:34:34",
    "component": "SasView",
    "summary": "Doc path breaks on Macs if exe is renamed",
    "priority": "minor",
    "keywords": "",
    "milestone": "SasView 3.1.0",
    "owner": "",
    "type": "defect"
}
```

**Comment by mdoucet** [comment]
Trac update at `2015/07/02 16:36:52`: 
* **mathieu** commented:

Just checked that this is no longer an issue since we fixed the documentation for v3.1.


* **mathieu** changed resolution from "" to "invalid"
* **mathieu** changed status from "new" to "closed"

**Comment by butlerpd** [comment]
Trac update at `2015/11/22 01:09:16`: **butler** changed milestone from "SasView Next Release +1" to "SasView 3.1.0"
